src/FacilityConformance/FacilityConformanceApp.cs

```diff
@@ -92,7 +91,7 @@ public async Task<int> RunAsync(IReadOnlyList<string> args)
 				});
 
 			await new WebHostBuilder()
-				.UseKestrel()
+				.UseKestrel(options => options.AllowSynchronousIO = serializerName is "newtonsoftjson")
```

Comment:
Is this needed with `ForceAsyncIO = true`?

Reply:
No, but that's only supported on the (now obsolete) `JsonHttpContentSerializer`.

Reply:
Oh, I see, it's the difference between `newtonsoftjson` and `obsoletejson`?

Reply:
Yes.